src/rest/Facet/FacetSortCriteria.ts

```diff
@@ -0,0 +1,14 @@
+export enum FacetSortCriteria {
+  score = 'score',
+  alphaascending = 'alphaascending',
+  alphadescending = 'alphadescending',
+  occurrences = 'occurrences'
+}
+
+export function isFacetSortCriteria(sortCriteria: string) {
+  return (
```

Comment:
Since typescript enums support reverse lookups, you could simply do:

```
export function isFacetSortCriteria(sortCriteria: string) {
    return FacetSortCriteria[sortCriteria] != null;
}
```

At the moment, the keys and values are identical, so forward and reverse lookups are the same, but if they were to diverge, the above logic should continue to work.